### ISSUE TITLE
RuntimeError: Input and output sizes should be greater than 0, but got input (H: 0, W: 1) output (H: 270, W: 512)

### ISSUE THREAD

**Issue by yt-hub** [steventan0110/ObjectDetectionRL#1]
Could you teach me how to solve this problem?
 File "/home/users/yt/ObjectDetectionRL-main/main.py", line 249, in <module>
    main(args)
  File "/home/users/yt/ObjectDetectionRL-main/main.py", line 166, in main
    agent.train()
  File "/home/users/yt/ObjectDetectionRL-main/module/agent.py", line 267, in train
    cur_image = self.update_observed_region(original_image, cur_box)
  File "/home/users/yt/ObjectDetectionRL-main/module/agent.py", line 183, in update_observed_region
    image, _box = transform(img_roi, [box])
  File "/home/users/yt/ObjectDetectionRL-main/util/transforms.py", line 53, in __call__
    img = torchvision.transforms.functional.resize(img, (self.H, self.W))
  File "/home/users/yt/.conda/envs/torch110/lib/python3.6/site-packages/torchvision/transforms/functional.py", line 421, in resize
    return F_t.resize(img, size=size, interpolation=interpolation.value, max_size=max_size, antialias=antialias)
  File "/home/users/yt/.conda/envs/torch110/lib/python3.6/site-packages/torchvision/transforms/functional_tensor.py", line 558, in resize
    img = interpolate(img, size=[new_h, new_w], mode=interpolation, align_corners=align_corners)
  File "/home/users/yt/.conda/envs/torch110/lib/python3.6/site-packages/torch/nn/functional.py", line 3731, in interpolate
    return torch._C._nn.upsample_bilinear2d(input, output_size, align_corners, scale_factors)
RuntimeError: Input and output sizes should be greater than 0, but got input (H: 0, W: 1) output (H: 270, W: 512)

**Comment by steventan0110** [comment]
Hi, are you using our code to do new experiments or just trying to replicate the results? Are you using the same dataset that we used? Can you provide more context (such as a printout of the shape of original_image and cur_box?)?

**Comment by yt-hub** [comment]
Yep, with a new experiment, the above problem is solved, but the new problem I encountered is that when I trained the policy network to validate it, one epoch outputs all the same action. And is it necessary to freeze the weights of the feature extraction network? If I don't freeze the weights, they will be out of memory, but I tried many ways to solve this problem.
